app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -712,13 +724,39 @@ export class Dispatcher {
     }))
   }
 
+  public setConflictsResolved(repository: Repository) {
+    return this.appStore._setConflictsResolved(repository)
+  }
+
+  public initializeRebaseProgress(
```

Comment:
I think these functions can use some docs since they form the API of rebase flow.

Reply:
Oops, I made a start on this last night but missed these. Addressed in 25e82b0ea26ad8f999fa89eb30b05300d06a816a